partials/servers.html

```diff
@@ -106,7 +106,8 @@ <h5 class="text-sm text-grey mt-1">Security:</h5>
 
               {% if def.bearerFormat() %}
                 <span class="font-bold no-underline bg-blue-light text-white text-xs uppercase rounded"
-                  style="height: 20px;font-size: 11px;padding: 3px;">Header: {{def.name()}}</span>
+                  style="height: 20px;font-size: 11px;padding: 3px;">Header: {{'Authorization' if def.type() === 'http' else def.name()}}
```

Comment:
when I look at [the spec](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#fixed-fields-21) to under the meaning of `bearerFormat` and `name` I say it is enough to fix it like this

```suggestion
                  style="height: 20px;font-size: 11px;padding: 3px;">Header: Authorization
```

Pretty hardcoded but look:
- `bearerFormat` is only ment for `http` type so this `def.type() === 'http'` is not needed as no other security object will have this field
- `name` is only for `httpApiKey` type so `http` type will never have it so this `else def.name()` is never going to happen

Yeah I just realized it during the review... sorry for the misleading comment under issue. Does it make sense?